pages/_app.js

```diff
@@ -8,6 +8,9 @@ import { ThemeProvider } from 'styled-components'
 import { getSession } from '../lib/auth/auth'
 import { Role } from '../server/services/authorize/role'
 import { IntlProvider } from 'react-intl'
+import * as Sentry from '@sentry/browser'
+
+Sentry.init({dsn: "https://e8e41423c54e44218441559971b0a9d4@sentry.io/5169743"});
```

Comment:
Is this value ok to be in a public repository?  if not we can inject using encrypted env var.

Reply:
oops! have removed the sentry kit in favor of raygun, but have made the same mistake with the raygun api key. I'm so used to working on private repos, need to slow down a bit!

Have replaced the value and rotated the api key.  We should inject these values :)